addons/gst-web/src/index.html

```diff
@@ -385,6 +386,9 @@
       </div>
     </v-app>
   </div>
+  <style>
+    [v-cloak] { display: none; }
```

Comment:
Include this style rule in the style tag of head section. Format it as following:
```
[v-cloak] {
      display: none; 
}